pkg/logs/config/endpoints.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package config
+
+type Endpoint struct {
+	APIKey string `mapstructure:"api_key"`
+	Logset string
```

Comment:
I wish we could avoid this, I think we can live without it now that we have index filters.

Reply:
+1 but it'll be easy to drop, and I don't think today we're fully ready

Reply:
adding a comment though